app.rb

```diff
@@ -71,14 +73,6 @@ def t(*args)
 Dir[File.dirname(__FILE__) + '/models/*.rb'].each { |file| require file }
 Dir[File.dirname(__FILE__) + '/presenters/*.rb'].each { |file| require file }
 
-
-$check_index_mapping_exists = defined?(RAKE_SEARCH) === nil || RAKE_SEARCH === false
-if $check_index_mapping_exists
-  # Ensure Elasticsearch index mappings exist, unless we are creating it in the rake search initialize
-  Comment.put_search_index_mapping
```

Comment:
@maxrothman: As discussed, added rake task `search:validate_index` to test that the 'content' alias exists, and that it has the correct field mappings and types.